Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml

```
@@ -30,3 +30,29 @@
   id: PaperFugitiveReport
   name: fugitive report
   description: An arrest warrant for a space fugitive sent from GALPOL.
+
+- type: entity
+  name: pharmacy prescription
+  parent: Paper
+  id: PaperPrecsription
+  description: 'A piece of paper to justify the reagents you detain on your person. Issued by the pharmacist.'
```

```suggestion
  description: 'A piece of paper to justify the reagents you carry on your person. Issued by the pharmacist.'
```
detain is a weird word